rc/tools/git.kak

```diff
@@ -160,13 +170,14 @@ define-command -params 1.. \
         if git "${@}" > /dev/null 2>&1; then
           printf %s "echo -markup '{Information}git $1 succeeded'"
         else
+          printf 'echo -debug Note: file paths are assumed to start in the `git_work_tree_top_level`\n' "$1"
+          printf 'echo -debug Note: mileage might vary when `git_work_tree_top_level` != kak workspace path\n' "$1"
```

Comment:
If `git_work_tree_top_level` does not deviate from `git rev-parse --show-toplevel`, then we don't need to print this note.

Tangentially related, perhaps we should print the full command that failed, like:

	printf "fail '%s'\n" "$(printf 'failed to run git -C %s%s' "$kak_opt_git_work_tree_top_level" "$(printf ' %s' "$@")" | sed "s/'/''/g")"

Reply:
Fixed, added an extra warning for the "out-of-work-tree" that autocompletion is unavailable and which directories relative paths are resolved against